batchflow/models/metrics/regression.py

```diff
@@ -0,0 +1,134 @@
+""" Contains regression metrics """
+import numpy as np
+
+from . import Metrics
+
+METRICS_ALIASES = {'mae': 'mean_absolute_error',
+                   'mse': 'mean_squared_error',
+                   'rmse': 'root_mean_squared_error',
+                   'r2': 'r2_score',
+                   'acc': 'accuracy'}
+
+class RegressionMetrics(Metrics):
+    """ Metrics to assess regression models
+
+    Parameters
+    ----------
+    targets : array-like of shape = (n_samples) or (n_samples, n_outputs) \
+              or list of such arrays
+        Ground truth (correct) target values.
+
+    predictions : array-like of shape = (n_samples) or (n_samples, n_outputs) \
+                  or list of such arrays
+        Estimated target values
+
+    multi: bool
+        Whether the task is multiouput (default is False)
+
+    weights : array-like of shape (n_samples)
+        Sample weights.
+
+    gap : float
+        Max difference between target and prediction for sample to be \
+        considered as properly classified (default is 3)
+
+    Notes
+    -----
+    - Accuracy metric for regression task stands for the ratio of samples for which `abs(targets-predictions) < gap`.
+
+    - For all the metrics, except max error, accuracy and median absolute error, you can compute sample-wise weighting.
+    For that purpose specify `weight` argument, which must be the same size as inputs.
+
+    In the multioutput case metrics might be calculated with or without outputs averaging.
+
+    Available methods are:
+
```

Comment:
please provide example and/or reference to base class docs how to use aggregation